transforms/.make.transforms

```diff
@@ -335,8 +335,9 @@ minio-stop:
 	if [ -e pyproject.toml ]; then	\
 	    dash_name=$$(echo $(TRANSFORM_NAME) | sed -e 's/_/-/g');		\
 	    cat pyproject.toml | sed 	\
-	    -e 's/"dpk-$(TRANSFORM_NAME)-transform-python\([=<>][=<>]\).*",/"dpk-$(TRANSFORM_NAME)-transform-python\1$(TRANSFORM_PYTHON_VERSION)",/'	\
-	    -e 's/"dpk-'$${dash_name}'-transform-python\([=<>][=<>]\).*",/"dpk-'$${dash_name}'-transform-python\1$(TRANSFORM_PYTHON_VERSION)",/'	\
+	    -e 's/"dpk_$(TRANSFORM_NAME)_transform_python\([=<>][=<>]\).*"/"dpk_$(TRANSFORM_NAME)_transform_python\1$(TRANSFORM_PYTHON_VERSION)"/'	\
```

Comment:
why not rename ones all dpk-...-transform-pyton ?

Reply:
Thanks and done.